texar/torch/modules/encoders/roberta_encoder.py

```diff
@@ -279,30 +239,8 @@ def forward(self,  # type: ignore
               character of the input (`CLS`), see RoBERTa's paper.
```

Comment:
Since RoBERTa encoder does not use `segment_ids`, let's mention it in the docstring.